script/autoapi/nettests-ffi.hpp.j2

```diff
@@ -0,0 +1,448 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
```

Comment:
Same as above: is it readable? (My comments are in the `nettests/ffi.hpp` header diff.)